src/common/string-utils.ts

```diff
@@ -1,3 +1,6 @@
+// eslint-disable-next-line import/no-unassigned-import
+import 'core-js/proposals/well-formed-unicode-strings';
```

Comment:
**suggestion (non-blocking):** Consider adding a comment with the explanation, so we know that it is safe to remove it when it is time :)

```ts
// TODO: Remove the import and `core-js` dependency once the local dev environment is updated to Node.js 20+.
// eslint-disable-next-line import/no-unassigned-import
import 'core-js/proposals/well-formed-unicode-strings';
```